bool mechanical_calibration
---
bool sanity_check